FileManager/Services/LocalNotificationsService.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  LocalNotificationsService.swift
+//  FileManager
+//
+//  Created by Alex on 20.07.22.
+//
+
+import Foundation
+import UIKit
+import UserNotifications
+
+class LocalNotificationsService {
+    static let shared = LocalNotificationsService()
+    
+    let notificationCenter = UNUserNotificationCenter.current()
+    
+    private init() {}
+    
+    func requestNotificationsPermissions() {
+        
+        notificationCenter
+            .requestAuthorization(options: [.alert, .badge, .sound]) { (granted, error) in
+                
+                guard granted else { return }
+                self.notificationCenter.getNotificationSettings { (settings) in
+                    print(settings)
+                    guard settings.authorizationStatus == .authorized else { return }
+                }
+            }
+        self.sendFarewellNotification()
+        self.sendRemindingNotification()
```

Comment:
You need to call this function after a user grants a permission (the 28th line). Your current way the notification will be added before a user permission is received

Reply:
Solved.)